azurecontainerapps.ts

```diff
@@ -511,6 +523,14 @@ export class azurecontainerapps {
                 this.commandLineArgs.push(`--env-vars ${environmentVariables}`);
             }
         }
+
+        if(!this.imageToDeploy.startsWith('default/')) {
```

Comment:
nit: let's make sure there's a space between the `if` and the opening `(` of the conditional

(same comment for the if-statement added a couple of lines below)